src/System.Security.Cryptography.Encoding/tests/Asn1/Reader/ReadBoolean.cs

```diff
@@ -99,7 +99,7 @@ public static void TagMustBeCorrect_Custom(PublicEncodingRules ruleSet)
             byte[] inputData = { 0x80, 1, 0xFF };
             AsnReader reader = new AsnReader(inputData, (AsnEncodingRules)ruleSet);
 
-            AssertExtensions.Throws<ArgumentException>(
+            Assert.Throws<ArgumentException>(
```

Comment:
Ah, sorry. I couldn't do simple find and replace because there were few overloads without the parameter name. And I forgot to press Save All in VS Code before doing the commit.